spec/unit/mutant/ast/regexp_spec.rb

```diff
@@ -70,9 +70,9 @@ def self.expect_mapping(regexp, type, &block)
 
         include_context 'regexp transformation'
 
-        return if regexp.encoding.name.eql?('ASCII-8BIT')
-
-        include_context 'regexp round trip'
+        unless regexp.encoding.name.eql?('ASCII-8BIT')
```

Comment:
@dkubb AFAIK only mutants spec suite has such places, regardless a constant lookup is more primitive than a lookup on a presentation (the string) of that constant. Very good catch.